content/python.rst

```diff
@@ -140,18 +140,23 @@ NO
   Unicode is a character set and UTF-8 is a particular way of
   encoding Unicode.
 
-* When a method does not use the *self* attribute, this is a code smell
-  that this method should be placed somewhere else.
+* When a method does not use the *self* attribute, this is a sure sign
+  that this method should be placed somewhere else. Also, it's a code smell.
 
 
-Abstract
-========
+Abstract classes - Twisted style
+================================
+
+The descriptions used below for abstract classes follows the 'definition' 
+used by the Twisted framework. While not exactly in line with cannonical Python
+definitions (that mention metaclasses and the use of decorators) these points
```

Comment:
we should still uses metaclasses and decorators when it make sense :)

and this is they styleguide for the Chevah project and not for Twisted